src/web/components/SharingPermission/ParticipantSearchBar.scss

```diff
@@ -35,8 +34,8 @@
   outline: none;
   color: var(--theme-button-text);
   font-weight: 400;
-  font-size: 1.875rem;
-  line-height: 2.25rem;
+  font-size: 1.1875rem;
+  line-height: 1.4375rem;
```

Comment:
Lionell recently suggested I not use rem for `line-height`, since it can start acting weirdly when the font size changes. e.g. with a really big font and a rem `line-height`, the text cuts off. The solution is to just use it without units, where it acts as a scalar, though you might need to play with the values to make it look the same. Is that okay?

Reply:
Good catch! I ran into the same problem before, will remove all units